src/components/common/SideBar/BoardList.tsx

```diff
@@ -0,0 +1,42 @@
+import { Link } from 'react-router-dom';
+import { BOARDS, Boards } from './constants';
+import clsx from 'clsx';
+
+interface BoardItemProps {
+  boardType: string;
+  isCurrent?: boolean;
+}
+
+function BoardItem({ boardType, isCurrent }: BoardItemProps) {
+  const { boardName, iconOn, iconOff, link } = BOARDS[boardType as keyof Boards];
```

Comment:
이 부분에서 '...expression of type string cannot be used to index...’ 타입스크립트 에러로 시간 많이 썼는데 [이 글](https://stackoverflow.com/questions/57086672/element-implicitly-has-an-any-type-because-expression-of-type-string-cant-b) 보고 바로 해결되었습니다. 나중에 혹시 같은 문제를 마주친다면 참고하십쇼ㅎ~ㅎ